cachi2/core/package_managers/pip.py

```diff
@@ -300,30 +300,37 @@ def _extract_metadata_from_config_files(
     1. pyproject.toml
     2. setup.py
     3. setup.cfg
```

Comment:
**Commit message:**

I don't see "breaking change" mentioned anywhere in the commit. This needs to be communicated clearly that this adjustment, while simplifying things, is a breaking change to some extent and it'll also have to be mentioned in the release notes.